test-framework/junit5/src/main/java/io/quarkus/test/junit/IntegrationTestUtil.java

```diff
@@ -330,6 +335,11 @@ public String networkId() {
             return networkId;
         }
 
+        @Override
+        public boolean manageNetwork() {
```

Comment:
Isn't this the same as checking if `networkId == null`?

Reply:
It is yeah

Reply:
No, actually it isn't - we only want to manage the network when it's not already done by test containers.